src/constants/facility.ts

```diff
@@ -0,0 +1,56 @@
+import { Facilities } from "../models/facility";
+import shineMoodCCData from "../assets/data/markers/facilities/shinemood-cc.json"; // 小木屋&校計中
+import eng3Eng4Data from "../assets/data/markers/facilities/eng3-eng4.json";
+import caCafe1Data from "../assets/data/markers/facilities/ca-cafe1.json";
+import parkingLotCafe2Data from "../assets/data/markers/facilities/parkinglot-cafe2.json";
+import swimMultiGymData from "../assets/data/markers/facilities/swim-multigym.json";
+import outdoorFieldData from "../assets/data/markers/facilities/outdoor-field.json";
+import oldGymData from "../assets/data/markers/facilities/oldgym.json";
+import libraryData from "../assets/data/markers/facilities/library.json";
+
+function getLocations(clusterName: string): Facilities {
+  let facilities: Facilities = {};
+  // let dataFromJson: DataFromJson = [];
+
+  switch (true) {
+    case /^小木屋&校計中/.test(clusterName):
+      facilities = shineMoodCCData;
```

Comment:
```suggestion
      return shineMoodCCData;
```

By doing so we do not need to maintain `facilities`.